src/gennylib/test/context_test.cpp

```diff
@@ -136,16 +136,19 @@ SchemaVersion: 2018-07-01
         std::atomic_int calls = 0;
         auto foobar = genny::testing::toDocumentBson("foo: bar");
 
+        auto fromDocListAssert = false;
         auto fromDocList = std::make_shared<OpProducer>([&](ActorContext& a) {
             for (const auto&& [k, doc] : a["docs"]) {
-                auto docgen = doc.to<DocumentGenerator>(a, 0);
-                REQUIRE(docgen().view() == foobar.view());
+                auto docgen = doc.to<DocumentGenerator>(a, 1);
+                fromDocListAssert = docgen().view() == foobar.view();
```

Comment:
I had to make this and several similar changes because Catch2 isn't thread-safe. I'd considered having a flag to make execution serial for tests, but figured it'd be best to exercise the parallelism.